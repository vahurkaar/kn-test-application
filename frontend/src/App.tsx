import React from 'react';
import PersonTable from './components/person-table/PersonTable';

const App = () => (
  <div className="container">
    <h1>KN Test</h1>

    <PersonTable />
  </div>
);

export default App;
