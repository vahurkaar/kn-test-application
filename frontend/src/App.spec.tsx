import App from './App';
import { render } from '@testing-library/react';
import React from 'react';

jest.mock('./components/person-table/PersonTable', () => 'person-table');

describe('App', () => {
  it('should render', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  })
});