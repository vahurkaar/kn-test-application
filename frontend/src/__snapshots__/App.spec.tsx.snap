// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      KN Test
    </h1>
    <person-table />
  </div>
</div>
`;
